plugins/package-managers/swiftpm/src/funTest/kotlin/SwiftPmFunTest.kt

```diff
@@ -31,30 +31,34 @@ import org.ossreviewtoolkit.utils.test.getAssetFile
 import org.ossreviewtoolkit.utils.test.matchExpectedResult
 
 class SwiftPmFunTest : WordSpec({
-    "Parsing 'Package.resolved' dependencies" should {
+    "Analyzing a lockfile with file format version 1" should {
         "return the correct result" {
-            val definitionFile = getAssetFile("projects/synthetic/spm-app/Package.resolved")
-            val expectedResultFile = getAssetFile("projects/synthetic/spm-expected-output-app.yml")
+            val definitionFile = getAssetFile("projects/synthetic/lockfile-v1/Package.resolved")
```

Comment:
Nit: To isolate this even more from "project-with-lockfile", maybe call this "lockfile-v1-*only*" (or "*only*-lockfile-v1")?

Reply:
I'd do this with a new commit (PR) on top, to avoid some hazzle with updating expected file, if this is ok for you.

Reply:
Addressed by https://github.com/oss-review-toolkit/ort/pull/8165.